js/ui/editors/CompileLogViewer.js

```diff
@@ -28,7 +28,9 @@ class KConstructorOutput extends ConstructorViewFileKind {
      * @param {Job} job
      */
     static getJobName = (job) => {
-        return `${job.projectDisplayName} - ${job.command}${job.stopped ? ' - Finished' : ''}`;
+        return `${job.projectDisplayName} - ${job.command}${
+            job.stopped ? (' - ' + (job.exitCode === -1 ? 'Stopped' : ((job.exitCode !== null && job.exitCode > 0) ? 'Failed' : 'Finished'))) : ''
```

Comment:
This one really needs to become some actual log - drawing the line at nested ternaries lol

Reply:
i didn't turn it into an actual log but i cleaned up the nested ternaries by making it into a property on the Job object